### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by rnystrom**
What do you think?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -9,6 +9,8 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the Software is
 furnished to do so, subject to the following conditions:
 
+THIS IS SOME TEXT I'M INSERTING INTO THE LICENSE
```
Comment (rynytest) [inline LICENSE:12]: This is a single comment

Comment (rnystrom) [reply LICENSE:12]: This is the OP's reply

File: LICENSE [hunk 2]
```diff
@@ -9,6 +9,8 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
...
+
```
Comment (rynytest) [inline LICENSE:13]: This is another review

File: LICENSE [hunk 3]
```diff
@@ -9,6 +9,8 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the Software is
```
Comment (rynytest) [inline LICENSE:9]: What about this?

**Review by rynytest — DISMISSED** [review]
This is my approve comment

**Review by rynytest — DISMISSED** [review]
Something something something

**Comment by rynytest** [comment]
@rnystrom ping

File: LICENSE [hunk 4]
```diff
@@ -9,6 +9,8 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
...
 furnished to do so, subject to the following conditions:
 
+THIS IS SOME TEXT I'M INSERTING INTO THE LICENSE
+
 The above copyright notice and this permission notice shall be included in all
```
Comment (rynytest) [inline LICENSE:14]: bla bla bla

**Review by rynytest — APPROVED** [review]
This is my comment on the review 🌮

**Pull request closed without merging**